image_recognition_msgs/srv/DetectPeople.srv

```diff
@@ -0,0 +1,5 @@
+# Interface that can request Face Properties of an array of face imaegs
+
+sensor_msgs/Image image
+---
+Person[] people
```

Comment:
I would just keep it with `DetectPersons` and name this variable `persons`, to me more formal, logical